src/grpc/request_template.rs

```diff
@@ -26,6 +26,7 @@ pub struct RequestTemplate {
     pub body: Option<Mustache>,
     pub operation: ProtobufOperation,
     pub operation_type: GraphQLOperationType,
+    pub protobuf_set: ProtobufSet,
```

Comment:
I see that to parse find type for error we are passing protobuf_set for every grpc related entity.

But basically we can get the DescriptorPool and thus find the message through ProtobufOperation by `method.parent_pool()`.
Although, it's little hacky it'll simplify the pr, wdyt?

Reply:
done